language_service/src/code_lens.rs

```diff
@@ -100,3 +96,30 @@ pub(crate) fn get_code_lenses(
         })
         .collect()
 }
+
+/// Uses the entry expression in the package to find the
+/// entrypoint callable item id. The entry expression has to match the
+/// specific pattern generated by the "generate entry expression" pass.
```

Comment:
I would just add a comment that provides a bit more information about what is the current pattern, something like:
> The auto-generated entry expression is a call to a parameterless function or operation.